src/pages/Wishlist/Wishlist.jsx

```diff
@@ -1,39 +1,52 @@
-import { products } from "../../backend/db/products";
+import { Link } from "react-router-dom";
 import { Listing } from "../../components";
-import { useDocumentTitle } from "../../customHooks";
+import { useWishlist } from "../../contexts";
+import { useAxios, useDocumentTitle } from "../../customHooks";
 import "./Wishlist.css";
 
 export const Wishlist = () => {
 	useDocumentTitle("Wishlist | MS");
+	const { axiosRequest } = useAxios();
+	const { wishlist, totalQty, wishlistDispatch, getUpdatedWishlist } =
+		useWishlist();
+
+	// useEffect(() => {
+	// 	(async () => {
```

Comment:
remove unwanted comment

Reply:
I have a plan of implementing it so kept.